src/Simulation/Simulation.jl

```diff
@@ -228,9 +228,8 @@ function Grid(sim::Simulation{T, Cylindrical};
                 for_weighting_potential::Bool = false,
                 max_tick_distance::Union{Missing, LengthQuantity, Tuple{LengthQuantity, AngleQuantity, LengthQuantity}} = missing,
                 max_distance_ratio::Real = 5,
-                add_ticks_between_important_ticks::Bool = true)::CylindricalGrid{T} where {T}
+                add_ticks_between_important_ticks::Bool = true, world = sim.world )::CylindricalGrid{T} where {T}
```

Comment:
That is passed twice as `sim` is the first argument.

Reply:
That is true, but this is the default only. When using a symmetry, another world will be passed, therefor I need this keyword